Makefile

```diff
@@ -176,12 +179,11 @@ deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in
 undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/config.
 	$(KUSTOMIZE) build config/default | kubectl delete -f -
 
-deploy-kcp: manifests install ## Install CRDs and deploy HAS on KCP
+deploy-kcp: ensure-tmp manifests kustomize ##install ## Install CRDs and deploy HAS on KCP
```

Comment:
should install be commented out?

Reply:
Yes, for now. The APIResources and APIExports manage the installation of the CRDs.